Entities/Items/Explosives/Keg.cfg

```diff
@@ -73,8 +73,7 @@ $name                                             = keg
 													ExplodeOnDie.as;
 													NoPlayerCollision.as;
 													BehindWhenAttached.as;
-													SetTeamToCarrier.as;
-													SetDamageToCarrier.as;
+													SetTeamToCarrier.as
```

Comment:
I may be wrong, but doesn't there need to be a semicolon after each script?

Reply:
I am wrong.

Reply:
not needed, but changed for consistency